packages/next/src/build/webpack/plugins/wellknown-errors-plugin/parse-dynamic-code-evaluation-error.ts

```diff
@@ -0,0 +1,22 @@
+import type { webpack } from 'next/dist/compiled/webpack/webpack'
+import { formatModuleTrace, getModuleTrace } from './getModuleTrace'
+import { SimpleWebpackError } from './simpleWebpackError'
+
+export function getDynamicCodeEvaluationError(
+  message: string,
+  module: webpack.NormalModule,
+  compilation: webpack.Compilation,
+  compiler: webpack.Compiler
+): SimpleWebpackError {
+  const { moduleTrace } = getModuleTrace(module, compilation, compiler)
+  const { formattedModuleTrace, lastInternalFileName, invalidImportMessage } =
+    formatModuleTrace(compiler, moduleTrace)
+
+  return new SimpleWebpackError(
```

Comment:
What's the reason to use simple webpack error here instead of simply appending the module trace to the error message?

Reply:
I followed the convention from other well-known errors. All other `parse*` return the same.